README.markdown

```diff
@@ -98,6 +98,12 @@ See the [documentation][docs] for arguments and [options].
 
     For more information on the `-Xmx` option, see the [Java SE Documentation][javadoc].
 
+3.  For potential contributions to the project, test the changes:
+
+        ./gradlew test             # run module unit tests
+        ./gradlew integrationtest  # run module integration tests
+        ./gradlew e2etest          # run end-to-end tests
+
```

Comment:
IMO these should not be under "Distribution builds" section. A more suitable place would be as the 5th step for "For developers" section.

Reply:
Silly error fixed.